index.html

```diff
@@ -31,7 +31,7 @@ <h4>I am a Software Developer</h4>
             <h2 id="heading">My Recent Works</h2>
             <article class="works">
                 <div class="picture" >
-                    <img src="./images/work1.jpg" alt="My First Project">
+                    <img src="./images/Img Placeholder.png" alt="My First Project">
```

Comment:
- [ ] Please, as the previous reviewer mentioned 🗡️, the image is not displayed, the problem is in the path, please replace with this line `<img src="./images/Work1.jpg" alt=" My First Project">` and make sure that the image has been displayed. 🍔 

![image](https://user-images.githubusercontent.com/91301423/198815348-2c4ad471-e343-4eb0-9a5b-be8b8cc7dbe3.png)

Reply:
Hi @fherrerao,

would you please check again? because I don't find this line (<img src="./images/Work1.jpg" alt=" My First Project">) in index file.

Thanks